src/views/scratch_1.4/scratch_1.4.jsx

```diff
@@ -121,7 +127,28 @@ const OnePointFour = () => (
                     <span className="nav-spacer" />
                     <h2><FormattedMessage id="onePointFour.faqsTitle" /></h2>
                     <h3><FormattedMessage id="onePointFour.resourcesQ" /></h3>
-                    <p><FormattedHTMLMessage id="onePointFour.resourcesA" /></p>
+                    <p>
+                        <FormattedMessage
+                            id="onePointFour.resourcesA"
+                            values={{
+                                gettingStartedGuide: (
+                                    <a href="http://download.scratch.mit.edu/ScratchGettingStartedv14.pdf">
```

Comment:
I think there is a localized version of this

Reply:
@apple502j I think that people have translated the 1.4 guide in the past, but it's never been systematically collected and shared on the Scratch website. So I think that's outside the scope for this PR, and probably not a priority right now.